tasks/push-snapshot/push-snapshot.yaml

```diff
@@ -24,7 +24,7 @@ spec:
     - name: retries
       description: Retry copy N times.
       type: string
-      default: "0"
+      default: "3"
```

Comment:
Interesting, wasn't aware this was even here and not used. Just to be clear: This means that we will now have up to 5*3=15 retries when something fails in the push-snapshot task. That sounds like a lot. But fine with me, if that's the intention.

Reply:
you are right. I removed the task retries

Reply:
but now there is no reason to upversion that task, so you need to remove that. sorry...